### DISCUSSION TITLE
Incorrect `copyright year` in the footer of website

### DISCUSSION THREAD

**Discussion by Abinashbunty** [executablebooks/meta#999] [General]
Hi team! Just spotted a small issue on the [website ](https://executablebooks.org/en/latest/)and wanted to report it here. 😄 

In the copyright section in footer, the year is wrong. Currently it is written as `© Copyright 202, Executable Book Project.` Anything like 2023 or 2022 etc would be fine. But a 3-digit year is not quite common.

**Comment by rowanc1** [comment]
Put together a PR to fix this here!
https://github.com/executablebooks/meta/pull/1044

Thank you!
